websites/W/Worldhideaway/presence.ts

```diff
@@ -2,151 +2,177 @@ const presence = new Presence({
+						{
+							label: "Viendo Miembros",
+							url: href,
+						},
+					];
+				} else {
+					presenceData.buttons = [
+						{
+							label: "Viendo Miembro",
+							url: href,
+						},
+					];
+					presenceData.largeImageKey = document
+						.querySelector('[alt*="Profile photo of"]')
+						.getAttribute("src");
+					presenceData.details = `Viendo ${
+						document.querySelector(
+							"#item-header-content > div > div > div.flex.align-items-center.member-title-wrap > h2"
+						)?.textContent
+					}'s Perfil`;
+				}
+				break;
+			}
+			case "category": {
+				presenceData.buttons = [
+					{
+						label: "Viendo Categoría",
+						url: href,
+					},
+				];
+				presenceData.details = "Viendo categoría";
+				presenceData.state = document.querySelector(
+					"#main > header > h1 > span"
+				).textContent;
+				break;
+			}
+			case "faqs": {
+				presenceData.buttons = [
+					{
+						label: "Leyendo FAQ",
+						url: href,
+					},
+				];
+				presenceData.details = "Leyendo FAQ";
 				presenceData.smallImageKey = "reading";
-			} else if (page === "/contacto/") {
-				presenceData.details = "Viendo:";
-				presenceData.state = "Contacto";
-			} else if (expage.includes("/wp-login.php?action=lostpassword")) {
-				presenceData.details = "Viendo:";
-				presenceData.state = "Contraseña Olvidada";
-			} else if (expage === "https://worldhideaway.com/wp-login.php")
-				presenceData.details = "Conectar";
-			else {
-				switch (page) {
-					case "/register/": {
-						presenceData.details = "Registro";
-						break;
-					}
-					case "/logros/": {
-						presenceData.details = "Viendo:";
-						presenceData.state = "Logros";
-
-						break;
-					}
-					case "/events/": {
-						presenceData.details = "Viendo:";
-						presenceData.state = "Events";
-
-						break;
-					}
-					default:
-						if (page.includes("/event/")) {
-							title = document.querySelector(
-								"#tribe-events-content > div.bs-event-heading > div.tribe-event-schedule-long > div.bs-tribe-events-single-heading > h1"
-							);
-							presenceData.details = "Viendo:";
-							presenceData.state = title.textContent;
-						} else {
-							switch (page) {
-								case "/equipo/": {
-									presenceData.details = "Viendo:";
-									presenceData.state = "Equipo";
-
-									break;
-								}
-								case "/texto/": {
-									presenceData.details = "Viendo:";
-									presenceData.state = "Texto";
-
-									break;
-								}
-								case "/gestos/": {
-									presenceData.details = "Viendo:";
-									presenceData.state = "Gestos";
-
-									break;
-								}
-								case "/stickers/": {
-									presenceData.details = "Viendo:";
-									presenceData.state = "Pegatinas";
-
-									break;
-								}
-								case "/salas/": {
-									presenceData.details = "Viendo:";
-									presenceData.state = "Salas";
+				break;
+			}
+			case "contacto": {
+				presenceData.buttons = [
+					{
+						label: "Viendo Contacto",
```

Comment:
The label is the text that shows up on the button. Its purpose is to perform an action so you should be using the infinitive form instead of the gerund (Ver Contacto)